django_auth_ldap/backend.py

```diff
@@ -112,6 +112,15 @@ def __getstate__(self):
             k: v for k, v in self.__dict__.items() if k not in ["_settings", "_ldap"]
         }
 
+    @property
+    def cache(self):
+        try:
+            return self._cache
+        except AttributeError:
+            cache_name = getattr(self.settings, "AUTH_LDAP_CACHE", "default")
+            self._cache = caches[cache_name]
```

Comment:
Is there some performance reason the cache the lookup? Or just habit.

Either way, this is re-implementing Django's `cached_property` co can use that instead.

Reply:
Saves an attribute lookup when we hide it behind the property - but yea - forgot about the cached property decorator. I'll update htat.